markets/perps-market/contracts/storage/AsyncOrder.sol

```diff
@@ -587,7 +587,7 @@ library AsyncOrder {
         // get initial margin requirement for the new position
         (, , runtime.newRequiredMargin, ) = marketConfig.calculateRequiredMargins(
             newPositionSize,
-            fillPrice
+            PerpsPrice.getCurrentPrice(marketId, PerpsPrice.Tolerance.DEFAULT)
```

Comment:
this result is used again on line 596. would be more efficient to assign it to a variable and then reuse it.